build.cake

```diff
@@ -358,7 +358,9 @@ Task("CreateLinuxPackages")
         Env = new string[] { 
             $"VERSION={nugetVersion}",
             $"BINARIES_PATH=/artifacts/OctopusTools.{nugetVersion}.linux-x64.extracted/",
-            "PACKAGES_PATH=/artifacts"
+            "PACKAGES_PATH=/artifacts",
+            "SIGN_PRIVATE_KEY",
```

Comment:
What happens if these aren't defined? Should we be checking they exist locally first?
What happens if this script is run on a dev machine (ie, without those keys available). Will it still work?

Reply:
I've added checks that display the detailed instructions if there's a problem.

For local development, we any GPG key will do, or comment out the parts that need it. I thought about making these optional, but we don't want to make it too easy to skip signing by accident in our production process.